### PR TITLE
Update deployment-guide-enrollment-mamwe.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a statement that MAM is deprecated and  After December 2022 customers will not be able to configure any new WIP policies for devices registered via MEM without enrollment.

reference #https://techcommunity.microsoft.com/t5/windows-it-pro-blog/announcing-the-sunset-of-windows-information-protection-wip/ba-p/3579282

fixes #https://github.com/MicrosoftDocs/memdocs/issues/3365

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/fundamentals/deployment-guide-enrollment-mamwe.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ This article provides recommendations on when to use MAM. It also includes an ov
 > [!TIP]
 > [!INCLUDE [tips-guidance-plan-deploy-guides](../includes/tips-guidance-plan-deploy-guides.md)]
 
+> [!IMPORTANT]
+> Mobile Application Management (MAM) is deprecated for Windows. After December 2022 customers will not be able to configure any new WIP policies for devices registered via MEM without enrollment. This does not apply to devices enrolled into MEM.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/deployment-guide-enrollment-mamwe.md:50]:
```suggestion
> Mobile Application Management (MAM) is deprecated for Windows. After December 2022 customers will not be able to configure any new Microsoft Information Protection (WIP) policies for devices registered via MAM without enrollment. This does not apply to devices enrolled in MAM.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MandiOhlinger Please review and signoff this PR. Thanks

**Comment by yogkumgit** [comment]
@MandiOhlinger Kindly update. Thanks

**Review by MandiOhlinger — APPROVED** [review]

**Review by MandiOhlinger — APPROVED** [review]

**Comment by MandiOhlinger** [comment]
@msbemba Thank you for making these changes. I made some other changes too. The PR is now merged.

**Pull request merged**